function assert(condition, reason) {
    if (!condition)
        throw new Error(reason);
}
